CameraOverlay/configs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "activeOverlay": "Overlay_Demo2.py",
+  "device": "primary",
```

Comment:
Do we need to hardcode this? It'll be overwritten each time anyway.

Reply:
Totally fine to just leave it as a default value.